src/components/maps/leafletMapExplorer.vue

```diff
@@ -766,9 +767,9 @@
                       <div class="flex items-center space-x-4" v-if="items.wind.speed">
                         <!-- Left Column -->
                         <div class="flex flex-col text-sm space-y-1 min-w-[100px]">
-                          <div>Depth: {{ items.water.depth }}</div>
-                          <div>Wind Speed: {{ items.wind.speed }} Kt</div>
-                          <div>Wind Direction: {{ items.wind.direction }}°</div>
+                          <div>Wind Speed: {{ speedFormatKnots(items.wind.speed) }}</div>
```

Comment:
The formatting is done in the items computed function.
https://github.com/xbgmsharp/vuestic-postgsail/blob/68fd44af006fc0af16dd0166d3da882a67bc12bf/src/components/maps/leafletMapExplorer.vue#L629

Reply:
Yes, but localized units are added here (m, kts, °).